articles/openstack/ostack-how-static-ip.md

```diff
@@ -0,0 +1,175 @@
+| binding_profile       | None                                                                     |
+| binding_vif_details   | None                                                                     |
+| binding_vif_type      | None                                                                     |
+| binding_vnic_type     | normal                                                                   |
+| created_at            | 2018-09-05T12:26:55Z                                                     |
+| data_plane_status     | None                                                                     |
+| description           |                                                                          |
+| device_id             |                                                                          |
+| device_owner          |                                                                          |
+| dns_assignment        | None                                                                     |
+| dns_domain            | None                                                                     |
+| dns_name              | None                                                                     |
+| extra_dhcp_opts       |                                                                          |
+| fixed_ips             | ip_address='10.1.1.10', subnet_id='fc333721-47c8-4445-871b-d261eb195b41' |
+| id                    | f6c6ace7-3e33-4159-a0ce-f9d70ba69bf2                                     |
+| mac_address           | fa:16:3e:4f:70:b6                                                        |
+| name                  | test-static-ip-1                                                         |
+| network_id            | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx                                         |
+| port_security_enabled | True                                                                     |
+| project_id            | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx                                         |
+| qos_policy_id         | None                                                                     |
+| revision_number       | 5                                                                        |
+| security_group_ids    | 01ea0a6f-1a39-4196-b798-47db89499379                                     |
+| status                | DOWN                                                                     |
+| tags                  |                                                                          |
+| trunk_details         | None                                                                     |
+| updated_at            | 2018-09-05T12:26:55Z                                                     |
++-----------------------+--------------------------------------------------------------------------+
+```
+### Launch an instance with the static IP attached
```

Comment:
4. Launch an instance with the static IP attached:

openstack server create --flavor t1.nano --port test-static-ip-1 --image cirros ukc-test --wait